archeryutils/targets.py

```diff
@@ -127,6 +127,21 @@ def __init__(
         self.native_dist_unit = Length.definitive_unit(native_dist_unit)
         self.indoor = indoor
 
+    def __repr__(self) -> str:
```

Comment:
I wonder if it would be better to write out in native units?\
A user usually calls `repr()` to check and get an understanding, in which case they'd probably be thinking in the native units they defined the target in, rather than in the underlying working units?

Happy to hear thoughts otherwise.

Reply:
Actually this does make much more sense. I wanted to avoid sprinkling more unit conversion logic around, but instead I factored it into some properties and was able to re-use it more in `Round.get_info` too.